orchestration/dap_orchestration/tests/test_solids.py

```diff
@@ -47,7 +46,8 @@ def mode():
         resource_defs={
             "beam_runner": noop_beam_runner,
             "refresh_directory": dap_orchestration.resources.test_refresh_directory,
-            "outfiles_writer": dap_orchestration.resources.test_outfiles_writer
+            "outfiles_writer": dap_orchestration.resources.test_outfiles_writer,
+            "api_token": ResourceDefinition.mock_resource()
```

Comment:
This can be a real API config from a file (ie a test.yaml analogous to the global.yaml you introduced);  no reason to have the mock resource here